nft_ingester/src/api/dapi/search_assets.rs

```diff
@@ -39,6 +39,9 @@ pub async fn search_assets<TPF: TokenPriceFetcher>(
     token_price_fetcher: Arc<TPF>,
     metrics: Arc<ApiMetricsConfig>,
 ) -> Result<AssetList, StorageError> {
+    if options.show_fungible {
```

Comment:
is it expected to replace the token_type filter value when the show_fungible value is set